README.md

```diff
@@ -35,23 +35,21 @@ In order to generate these bindings from scratch, you will need to clone two dep
 
 **rust-lightning**, (a specific branch built for bindings compatibility)
 ```shell
-git clone --branch 2021-03-java-bindings-base https://github.com/TheBlueMatt/rust-lightning /path/to/rust-lightning
+git clone --branch 2022-06-108-java-bindings https://github.com/TheBlueMatt/rust-lightning ./bindings/artifacts/rust-lightning
```

Comment:
the path doesn't need to b ./bindings/artifacts, because it's specified in the environment variables. I would leave it as it was, even though it so happens to be the path we're using in the build action

Reply:
I messed up my environment variables enough times to appreciate having some nice defaults